custom_components/alexa_media/__init__.py

```diff
@@ -255,6 +258,8 @@ async def login_success(event=None) -> None:
                 "media_player": {},
                 "switch": {},
                 "sensor": {},
+                "temperature": [],
```

Comment:
We have historically tried to store only HA recognized sensors. Can this be easily moved to sensor? Admittedly it may complicate things since we'll have to check the device_class to determine what sensor it's for.

Reply:
I took a shot at this, but as expected it got complicated quickly. The issue wasn't actually the device_class, but the fact that the sensor map is keyed based on the media player id and the temperature sensor doesn't really have a media player id.

If you think the rule about HA recognized sensors is important, I think the best way forward is to change the structure here to be "sensor": {"temperature": [], "media_player_attached": {}}. Should I move forward with that?

Reply:
Interesting. So the temperature sensor isn't obviously attached to the relevant Alexa device serial? 

I don't think we need the second level key. If the sensor has a unique id, it can be used as a key in the sensor dict mapping to the entity. While right now we're storing notifications in the sensor dict, those make sense cause they're linked to a media_player. 

So structurally:
```json
"sensor": { 
"SERIAL_NUMBER": { "Alarm" :{"HA_ENTITY_1"}, "Timer": {}, "Reminder":{}},
"OTHER_UNIQUE_IDENTIFIER": {
  "whatever makes sense for that identifier": "HA_ENTITY_2"
 }
}
```
Right now we don't enumerate across all keys in the entities/sensor dict so it shouldn't hurt to add unique keys to it.

Reply:
I dove into the json a little more. It appears the device serial id was hiding in there for the echo temperature sensor. I used that. Sadly, I don't think there is actually any guarantee that it will link up to a media player so there is some small chance a new device serial may get introduced solely for a temperature sensor. I checked through all the code and verified that shouldn't break anything if it does happen.